config/samples/hstreamdb.yaml

```diff
@@ -33,6 +48,17 @@ spec:
     replicas: 1
     container:
       name: admin-server
+  console:
+    image: hstreamdb/hstream-console
+    imagePullPolicy: IfNotPresent
+    replicas: 1
```

Comment:
What happened if console replicas is greater than 1 ?

Reply:
Nothing. The console component is a stateless service.